ocaml/tests/dune

```diff
@@ -27,3 +27,7 @@
   )
   (action (run ./check-no-xenctrl %{x}))
 )
+
+(env
+  (_ (env-vars (XAPI_TEST 1)))
```

Comment:
`(_` looks like a typo! I assume it's correct though

Reply:
`_` is a wildcard for the environment name, IIRC